src/createAnimatedComponent.js

```diff
@@ -216,6 +216,19 @@ export default function createAnimatedComponent(Component) {
     }
 
     _attachPropUpdater() {
+      if (Platform.OS === 'ios') {
```

Comment:
why is this here? Seem like it'd belong better in place where we already set viewTag for reanimated2, that is a couple of lines lower in _attachAnmatedStyles

Reply:
fixed c1d13d9